h_malloc.c

```diff
@@ -1087,6 +1087,29 @@ COLD static void handle_bugs(void) {
         ro.purge_slabs = false;
         ro.region_quarantine_protect = false;
     }
+
+    char cmdline[256];
+    if (readlink("/proc/self/cmdline", cmdline, sizeof(cmdline)) == -1) {
+        return;
+    }
+
+    const char* broken_apps[] = {
```

Comment:
Type should be `static const char *const broken_apps[]` to make the array into a constant.